deploy/Datadog.Trace.ClrProfiler.WindowsInstaller/Product.wxs

```diff
@@ -151,7 +151,6 @@
     <ComponentGroup Id="EnvironmentVariables.Machine" Directory="INSTALLFOLDER">
       <Component Id="EnvironmentVariablesShared" Guid="{C314A305-9C24-4E46-9ECF-E5EEA703BDEA}" Win64="$(var.Win64)">
         <CreateFolder/>
```

Comment:
I'm removing `DD_INTEGRATIONS` from our MSI since we can key off `DD_DOTNET_TRACER_HOME`. If for some reason the user has a different integrations.json file they want to use, they can manually set `DD_INTEGRATIONS` to prioritize that one.